### DISCUSSION TITLE
What Microarchitecture Feature Level does al2023 target?

### DISCUSSION THREAD

**Discussion by wallentx** [amazonlinux/amazon-linux-2022#590] [Q&A] [answered]
For the x86_64 flavors of al2023, what feature level is the default compilation target that packages are built from?
I.e.
• `x86-64`: CMOV, CMPXCHG8B, FPU, FXSR, MMX, FXSR, SCE, SSE, SSE2
• `x86-64-v2`: (close to Nehalem) CMPXCHG16B, LAHF-SAHF, POPCNT, SSE3, SSE4.1, SSE4.2, SSSE3
• `x86-64-v3`: (close to Haswell) AVX, AVX2, BMI1, BMI2, F16C, FMA, LZCNT, MOVBE, XSAVE
• `x86-64-v4`: AVX512F, AVX512BW, AVX512CD, AVX512DQ, AVX512VL

**Comment by danie-dejager** [comment, marked as answer]
x86-64-v2:
https://docs.aws.amazon.com/linux/al2023/ug/system-requirements.html
